stream-configuration/src/main/java/com.backbase.stream.config/BackbaseStreamConfigurationProperties.java

```diff
@@ -54,6 +54,11 @@
          * Location of Limits Presentation Service.
          */
         private String limitsPresentationBaseUrl;
+
+        /**
+         * Location of Product Summary Service"
+         */
+        private String productSummaryBaseUrl;
```

Comment:
This one is not used anywhere, same as `arrangementPandpBaseUrl` and with lean services will have the same value of `accountPresentationBaseUrl` (arrangements-service). Maybe we should rename the latest to `arrangementsServiceBaseUrl`?